beta-src/src/utils/map/getOrdersMeta.ts

```diff
@@ -21,7 +21,7 @@ export default function getOrdersMeta(data, phase): Props {
         updateOrdersMeta[id] = {
           saved: true,
           update: {
-            type: toTerrID ? type : "Wait",
+            type: toTerrID ? type : "Destroy",
```

Comment:
Doesn't "Destroy" have a `toTerrID` specified? The "Wait" command here is intentional so it cannot be deleted as it will affect builds.